src/store/actions/async/index.js

```diff
@@ -19,3 +19,33 @@ const fetchHomePage = (url) => {
 // In this case, something like dispatch(fetchHomePageAction('http://ah-backend-summer-staging.herokuapp.com/'))
 // It uses redux thunk underneath and will dispatch the appropriate actions when the call returns
 export const fetchHomePageAction = createActionThunk(FETCH_HOME_PAGE_ACTION, fetchHomePage);
+
+export function getToken(localStorage) {
+    const token = localStorage.getItem('token');
+    if (!token || token.length < 1) {
+        throw "Not logged in";
+    }
+    return token;
+}
+
+export function getLoggedInUser(localStorage) {
```

Comment:
I know we had discussed on following conventions here for using arrow functions. Is there a specific reason as to why you decided to use ES5 functions